go/teams/chain.go

```diff
@@ -363,6 +364,33 @@ func (t *TeamSigChainState) checkSubteamCollision(id keybase1.TeamID, name keyba
 	return nil
 }
 
+func (t *TeamSigChainState) HasActiveInvite(name, typ string) (bool, error) {
+	i, err := t.FindActiveInvite(name, typ)
+	if err != nil {
+		if _, ok := err.(libkb.NotFoundError); ok {
+			return false, nil
+		}
+		return false, err
+	}
+	if i != nil {
+		return true, nil
+	}
+	return false, nil
+}
+
+func (t *TeamSigChainState) FindActiveInvite(name, typ string) (*keybase1.TeamInvite, error) {
+	ityp, err := keybase1.TeamInviteTypeFromString(typ, false)
+	if err != nil {
+		return nil, err
+	}
+	for _, invite := range t.inner.ActiveInvites {
+		if invite.Name == keybase1.TeamInviteName(name) && reflect.DeepEqual(invite.Type, ityp) {
```

Comment:
If that status quo really bothers you, then you can make `keybase1.TeamInviteType` a string as you want, and then make functions like `TeamInvitetype#IsKeybase()`, `TeamInviteType#IsEmail()` or `TypeInviteType#IsSBS()` instead of the current `variant`.  I'm fine with that too.